docs/using-the-app.md

```diff
@@ -24,8 +24,22 @@ The mirror should now be ready for PRs!
 
 1. Create a pull request against the private mirror repo
 
-2. Review and merge the PR - at this step, if issues come up that require the deletion of code in the PR, you should "squash history" in the PR so the merge commit does not contain traces of the unwanted code.
+2. Review and merge the PR - at this step, if issues come up that require the deletion of code in the PR, you should "squash history" in the PR so the merge commit does not contain traces of the unwanted code. Alternatively, you should rewrite any commits that include sensitive data with the sensitive data removed. Review the [security implications for contributions synced to public forks](#security-implications-for-contributions-synced-to-public-forks) for more information.
 
 3. Once it's merged, the app will automatically sync the private mirror to your public fork, updating a branch whose name matches the name of the private mirror (`silverteam-website`, in this example).
 
 4. You can then use that branch of the public fork as the base to open a PR to the upstream repository.
+
+## Security implications for contributions synced to public forks
+
+Private Mirrors App is designed to help you manage your open source contributions in a way that addresses stakeholder concerns and conforms to your organization's security and compliance requirements. Contributions using PMA are first made to a private mirror, which is detached from the repository and forks (i.e. the [repository network](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/understanding-connections-between-repositories)) on GitHub. This allows sensitive data or credentials to be caught in a private mirror before any commits are synced back to a public fork. Any sensitive data or credentials should be removed from history before syncing the contribution back to a public fork.
```

Comment:
You may also consider linking through to this GH wiki page (https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository) that describes how to remove the sensitive data from git history in this line:
> Any sensitive data or credentials should be removed from history before syncing the contribution back to a public fork.

Reply:
Thanks for the poinnter @coffeesippa but in fact, the procedure described on that page is not needed for Private Mirrors - one of the benefits of using the app is that commits on the mirror that get `squashed` down prior to merging will not be accessible on the public fork.

Reply:
Does Private Mirrors always squash down or is that configurable? From somewhere else in the docs I thought that was an option but not how PMA always worked.

Reply:
You're right it's not automatic - but the point is the final state of the commits that are pushed up to the public fork is all you'll see. (Unlike working strictly on a public fork where the `filter-repo` workflow is necessary to make sure that commits are not still reachable)

Reply:
Added some more explicit instructions for squashing or rewriting history in https://github.com/github-community-projects/private-mirrors/pull/232/commits/c0aa4819ae23a8e974862e7e015b48ded9b74666, let me know what y'all think